index.js

```diff
@@ -1,88 +1,121 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
+console.log("%cQuestion 1", "color: red") 
+let counter = 1;
 
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+counter++;
```

Comment:
this does what we want in that it changes the value to 2 (the first time around), but really all you had to do was change the value:  `counter = 2`